src/plugins/gitresolver/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+include (LibAddMacros)
+#if (DEPENDENCY_PHASE)
+    #    find_package(LibGit2)
```

Comment:
find_package(LibGit2) doesn't seem to work, any suggestion what to do ? (i'm not really good with cmake)